modules/vba_documents/app/workers/vba_documents/slack_notifier.rb

```diff
@@ -27,6 +29,7 @@ def perform
 
     def fetch_settings
       @slack_url = Settings.vba_documents.slack.notification_url
+      @internal_slack_url = Settings.vba_documents.slack.internal_notification_url
```

Comment:
Well, you have shined a light on my bad name of @slack_url.  I like your name invalid_parts_alerted.

Perhaps a rename:
@slack_url => @default_alert_url
@internal_slack_url => @invalid_parts_alert_url